src/main/java/com/example/music/backend/common/security/CustomUserPrincipal.java

```diff
@@ -0,0 +1,60 @@
+package com.example.music.backend.common.security;
+
+import com.example.music.backend.user.domain.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class CustomUserPrincipal implements UserDetails {
+
+    @Value("${one.user.one.role.capacity}")
+    private int ONE_USER_ONE_ROLE_CAPACITY;
```

Comment:
Why do you need ONE_USER_ONE_ROLE_CAPACITY ?

Reply:
deleted